libs/hdf-converters/src/gosec-mapper.ts

```diff
@@ -22,11 +22,29 @@ function nistTag(input: Record<string, unknown>): string[] {
   return CWE_NIST_MAPPING.nistFilter(cwe, DEFAULT_NIST_TAG);
 }
 
+// Check `nosec` and `suppressions` fields which denote whether the gosec rule violation should be suppressed/skipped
+function formatStatus(input: Record<string, unknown>): string {
+  return `${_.get(input, 'nosec')}` === 'false' &&
+    `${_.get(input, 'suppressions')}` === 'null'
+    ? ExecJSON.ControlResultStatus.Failed
+    : ExecJSON.ControlResultStatus.Skipped;
+}
+
+// If a gosec rule violation is suppressed, forward the given justification
+function formatSkipMessage(input: Record<string, unknown>): string {
+  return `${_.get(input, 'suppressions')}` !== 'null'
+    ? `${_.get(input, 'suppressions[0].justification')}`
+    : '';
```

Comment:
should we provide some string explicitly stating that a reason wasn't provided?

Reply:
Something like `N/A`?

Reply:
nah i would say something like "User didn't supply a reason" or something like that.

Reply:
Addressed.